src/packer.pkr.hcl

```
@@ -151,11 +151,162 @@ build {
   provisioner "powershell" {
     # Wait 90 seconds before executing the check-defender.ps1 powershell script.
     # This gives a generous grace period between restarting Windows and running the second provisioner.
```

Update comment to reflect shorter pause:
```suggestion
    # Wait 30 seconds before executing the next provisioner.
    # This gives a grace period between restarting Windows and running the next provisioner.
```